events/moose/moose.go

```diff
@@ -243,6 +243,15 @@ func (s *Subscriber) NotifyRate(data events.ServerRating) error {
 	))
 }
 
+func (s *Subscriber) NotifySendInvitation(any) error {
+	return s.response(moose.NordvpnappSendUserInterfaceUiItemsClick(
+		"send_invitation",
+		moose.NordvpnappUserInterfaceItemTypeTextBox,
```

Comment:
There are following UI item types available:

	// Simple button with text or icon.
	NordvpnappUserInterfaceItemTypeButton NordvpnappUserInterfaceItemType = 1
	// Check boxes that allow to select multiple items.
	NordvpnappUserInterfaceItemTypeCheckBox NordvpnappUserInterfaceItemType = 2
	// A drop-down menu that allows to select a single item or multiple items.
	NordvpnappUserInterfaceItemTypeDropdown NordvpnappUserInterfaceItemType = 3
	// A hyperlink that redirects user to an external location
	NordvpnappUserInterfaceItemTypeHyperlink NordvpnappUserInterfaceItemType = 4
	// Check boxes that only allow to select a single item.
	NordvpnappUserInterfaceItemTypeRadioButton NordvpnappUserInterfaceItemType = 5
	// Can be discrete (categorical, countable like "A", "B", "C") or continuous (from 0 to 10, like 0.4564565, 1.25435 etc.).
	NordvpnappUserInterfaceItemTypeSlider NordvpnappUserInterfaceItemType = 6
	// a.k.a toggle; allows binary selection (usually used in settings, E.g. auto connect -> on/off).
	NordvpnappUserInterfaceItemTypeSwitch NordvpnappUserInterfaceItemType = 7
	// A tab or screen selector item or a pop-up.
	NordvpnappUserInterfaceItemTypeTabOrScreen NordvpnappUserInterfaceItemType = 8
	// A field where user can enter some text; E.g. search box.
	NordvpnappUserInterfaceItemTypeTextBox NordvpnappUserInterfaceItemType = 9

So i think that `NordvpnappUserInterfaceItemTypeTextBox` fits the best, because user has to type in an email to send invitation in the command line.